### ISSUE TITLE
Doc Improvement

### ISSUE THREAD

**Issue by kaimo455** [tensorflow/tensorflow#53499]
https://github.com/tensorflow/tensorflow/blob/c256c071bb26e1e13b4666d1b3e229e110bc914a/tensorflow/python/eager/backprop.py#L812

dz_dy = tape.gradient(z, w)
The Variable `dz_dy` could be more intuitive by renaming to `dz_dw`.

**Comment by mohantym** [comment]
Hi @chunduriv ! Could you please look at this issue?

**Comment by chunduriv** [comment]
@kaimo455,

The issue will move to closed status once the [PR](https://github.com/tensorflow/tensorflow/pull/53518) is merged. Thanks!
